modules/weather.md

```diff
@@ -155,6 +155,9 @@ Please note Darksky no longer accept new signups and the API will shutdown at th
 | `lat`                | The latitude coordinate for the desired location. <br><br> **Possible value:** `59.322665376` <br> This value is **REQUIRED**                                                  |
 | `lon`                | The longitude coordinate for the desired location. <br><br> **Possible value:** `18.069666388` <br> This value is **REQUIRED**                                                 |
 | `precipitationValue` | The type of precipitation to display (min, max, median, mean). <br><br> **Possible values:** `'pmin'` , `'pmean'` , `'pmedian'`, `'pmax'`. <br> **Default value:** `'pmedian'` |
+| `location` | The location used for weather information. <br><br> **Example:** `'Stockholm'` <br> **Default value:** `false` <br><br> |
```

Comment:
Is there an URL where one can look up all available locations? That would be helpful here :-)

Reply:
This is a display name only, I used the name `location` because that [seems to be what has been used previously](https://github.com/SkySails/MagicMirror-Documentation/blob/add-location-config-option-smhi/modules/weather.md?plain=1#L93).

Should I add a short clarification about this?

Reply:
Yeah, that may be helpful, something like "The name displayed on top of the weather informaion"

Reply:
Sorry for the delay! Is that better?

Reply:
Yes, thx!